### DISCUSSION TITLE
支持下载快照版MC

### DISCUSSION THREAD

**Discussion by pybilibilimusic** [Meloong-Git/PCL#1097] [提问] [answered]
看其他主播测评，又不想花钱去买

**Comment by pybilibilimusic** [comment]
注：我是个中学生，家长不让花钱

**Comment by WForst-Breeze** [comment, marked as answer]
下载最新版 PCL2 后，前往 下载-自动安装-预览版，列出的所有版本即为快照版😏

既然帮忙就帮到底吧qwq（小声bb
补充：一般下拉框中最上面的那个即为最新版快照，点击下载即可，如果下不下来可以前往 设置-启动器-下载-版本列表获取 中选择 尽量使用镜像源，如果没看到你想要的最新快照就选择 尽量使用官方源 即可。
